misc/qutebrowser.appdata.xml

```diff
@@ -40,4 +40,7 @@
 	<url type="help">https://qutebrowser.org/doc/help/</url>
 	<url type="bugtracker">https://github.com/qutebrowser/qutebrowser/issues/</url>
 	<url type="donation">https://github.com/qutebrowser/qutebrowser#donating</url>
+    <releases>
+        <release version="1.1.0" date="2018-01-15" />
```

Comment:
1.1.1 was just released, bumping this number should probably be added to the release checklist (but I'm not sure how)